apps/frontend/src/app/3_organisms/FastBtcDialog/constants.ts

```diff
@@ -0,0 +1,13 @@
+// https://github.com/DistributedCollective/FastBTC/blob/development/config/config-main.js#L15
+export const DEPOSIT_FEE_SATS = 6000;
+
+// hardcoded 0.2% dynamic fee
+// https://github.com/DistributedCollective/FastBTC/blob/development/controller/rskCtrl.js#L64
+export const DEPOSIT_FEE_DYNAMIC = 20;
+
+// https://github.com/DistributedCollective/bidirectional-fastbtc/blob/master/packages/fastbtc-contracts/contracts/FastBTCBridge.sol#L105
+export const DYNAMIC_FEE_DIVISOR = 10000;
+
+export const Bitcoin = 'BTC';
```

Comment:
```suggestion
//TODO: refactor into Bitcoin asset definition later
export const Bitcoin = 'BTC';
```

We should really make Bitcoin asset definition but it's not going to be used anywhere else for the moment, just add a comment